README.rst

```diff
@@ -99,6 +99,11 @@ Quickstart
     docs.register(get_pets)
     docs.register(PetResource)
 
+Documentation
+-------------
+
+https://flask-apispec.readthedocs.org/
```

Comment:
`readthedocs.io` is now the preferred domain to use. Would you mind updating this?